tests/test.py

```diff
@@ -1,98 +0,0 @@
-import kubectl
```

Comment:
why this file deleted? instead of move to e2e folder?

Reply:
I moved all files at a time, but for some reason `test.py` was shown as deleted

Reply:
I can create a new PR to properly move `test.py` and `test_operator.py`